python/rpdk/java/codegen.py

```diff
@@ -345,6 +359,19 @@ def generate(self, project):
             package_name=self.package_name,
             operations=project.schema.get("handlers", {}).keys(),
             pojo_name="ResourceModel",
+            wrapper_parent="LambdaWrapper",
+        )
+        project.overwrite(path, contents)
+
+        # write generated handler integration with ExecutableWrapper
+        path = src / "ExecutableHandlerWrapper.java"
```

Comment:
Generate this new class will cause build break if customer doesn't update the java plugin version.  And it happens quite common as update java plugin needs to update the pom.xml. 

One way we use during 2.0 upgrade is to check the java plugin version, and give an error if doesn't match.  I think you may can leverage the `_get_java_plugin_dependency_version` to check only generate this class if version is 2.1.0 or above. Assuming we will bump the java plugin version to 2.1.0.

Reply:
Good call out. Current version is 2.0.2 so I just said anything greater than 2.0.3 so we can bump to that or 2.1